robot/basestation/app.py

```diff
@@ -245,6 +247,36 @@ def capture_image():
 
     return jsonify(msg=msg)
 
+# This is called on if ffmpeg isn't already recording when the record toggle button is clicked
+@app.route("/start_recording_feed/<stream>", methods=["POST", "GET"])
+def start_recording_feed(stream):
```

Comment:
This function has many responsibilities. I would recommend making another function about starting and stopping ffmpeg.

Reply:
Aren't they already separate functions? Or do you want an seperate function from the one that makes variables, that would handle starting the stream?